src/Assetic/Filter/CJSMinFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Assetic\Filter;
+
+use Assetic\Asset\AssetInterface;
+use Assetic\Exception\FilterException;
+
+/**
+ * Runs assets through the C jsmin binary.
+ *
+ * @link http://javascript.crockford.com/jsmin.html
+ * @author Felix Yeung
+ */
+class CJSMinFilter implements FilterInterface
+{
+    private $jsminBin;
+
+    public function __construct($jsminBin = '/usr/bin/jsmin')
+    {
+        $this->jsminBin = $jsminBin;
+    }
+
+    public function filterLoad(AssetInterface $asset)
+    {
+    }
+
+    public function filterDump(AssetInterface $asset)
+    {
+        $assetFile = $asset->getSourceRoot() . '/' . $asset->getSourcePath();
+
+        $descriptorspec = array(
+            0 => array('file', $assetFile, 'r'),
+            1 => array('pipe', 'w'),
+            2 => array('pipe', 'w')
+        );
+
+        $process = proc_open($this->jsminBin, $descriptorspec, $pipes);
```

Comment:
Please use the Process component rather than using proc_open directly (which is very tricky to make work cross-platform because it has lots of quirks)